hs_core/tasks.py

```diff
@@ -374,6 +382,44 @@ def send_over_quota_emails():
             logger.debug('user ' + u.username + ' does not have UserQuota foreign key relation')
 
 
+@celery_app.task(ignore_result=True, base=HydroshareTask)
+def check_geoserver_registrations(resources):
+    # Check to ensure resources have updated web services registrations
+    if not resources:
+        cuttoff_time = timezone.now() - timedelta(days=1)
+        resources = BaseResource.objects.filter(updated__gte=cuttoff_time, raccess__public=True)
+
+    failed_resources = {}
+    err_msg = ""
+    try:
+        for res in resources:
+            response = update_web_services(
```

Comment:
The only other feedback I have is that these network calls happen sequentially.  It is only really a concern for large batches of resources.  It may not be much of a concern because of geoserver being in the same network as well.

Reply:
thanks @sblack-usu. I took the approach of distributing these network calls over 2 hours in https://github.com/hydroshare/hydroshare/pull/5186/commits/931d8ee39ea505e6947eb1651cb89d8a4b28895a